src/functionalTest/java/uk/gov/hmcts/reform/draftstore/endpoint/v3/GetAllTest.java

```diff
@@ -29,18 +30,14 @@
     @Autowired
     private MockMvc mockMvc;
 
-    @MockBean private DraftStoreDAO draftRepo;
-    @MockBean private AuthService authService;
+    @MockBean private DraftService draftService;
+    @MockBean private AuthService authService; //NOPMD
```

Comment:
no pmd?

Reply:
pmd complains about unused field, but it has to be there, we need a mock.

Reply:
would it still work if NOPMD was followed by explanation?

Reply:
done